cmd/root_test.go

```diff
@@ -201,6 +202,23 @@ func TestNewCommandArguments(t *testing.T) {
 				}},
 			}),
 		},
+		{
+			desc: "using the private-ip flag",
+			args: []string{"--private-ip", "proj:region:inst"},
+			want: withDefaults(&proxy.Config{
+				PrivateIP: true,
+			}),
+		},
+		{
+			desc: "using the private-ip flag query param",
+			// the query param's presence equates to true
```

Comment:
This is an old comment and can be deleted.

Reply:
done.